bezier-rs/lib/src/lib.rs

```diff
@@ -272,52 +307,62 @@ impl Bezier {
 
 	/// Return an approximation of the length of the bezier curve.
 	pub fn length(&self) -> f64 {
-		// Code example from <https://gamedev.stackexchange.com/questions/5373/moving-ships-between-two-planets-along-a-bezier-missing-some-equations-for-acce/5427#5427>.
-
-		// We will use an approximate approach where
-		// we split the curve into many subdivisions
-		// and calculate the euclidean distance between the two endpoints of the subdivision
-		let lookup_table = self.compute_lookup_table(Some(LENGTH_SUBDIVISIONS));
-		let mut approx_curve_length = 0.0;
-		let mut prev_point = lookup_table[0];
-		// calculate approximate distance between subdivision
-		for curr_point in lookup_table.iter().skip(1) {
-			// calculate distance of subdivision
-			approx_curve_length += (*curr_point - prev_point).length();
-			// update the prev point
-			prev_point = *curr_point;
-		}
+		match self.handles {
+			BezierHandles::Linear => self.start.distance(self.end),
+			_ => {
+				// Code example from <https://gamedev.stackexchange.com/questions/5373/moving-ships-between-two-planets-along-a-bezier-missing-some-equations-for-acce/5427#5427>.
+
+				// We will use an approximate approach where
+				// we split the curve into many subdivisions
```

Comment:
I split it up because the description is the beginning of the implementation, while the comment with the link is referring to the entire function. It didn't seem right to have the in the same section and hence making them appear as tightly coupled comments.

Reply:
Just make the two short comment lines into one. The previous comment with the link is fine as-is.